packages/network-controller/src/NetworkController.test.ts

```diff
@@ -3722,7 +3507,9 @@ describe('NetworkController', () => {
     it('returns the EthQuery object set after the provider is set', async () => {
       const messenger = buildMessenger();
       await withController({ messenger }, async ({ controller }) => {
-        const fakeEthQuery = {};
+        const fakeEthQuery = {
+          sendAsync: jest.fn(),
```

Comment:
I'm not sure this is the best approach, but we need this to be a function now, or `#getLatestBlock()` will raise an unhandled promise rejection.

Reply:
Oops, I guess I got away with `sendAsync` not being present before! This makes sense.